jupyterhub/bases/custom-images/jupyterhub-custom-images.yaml

```diff
@@ -357,6 +357,42 @@ spec:
     importPolicy:
       scheduled: true
     name: "latest"
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  name: s2i-pytorch-py38-notebook
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - from:
+      kind: DockerImage
+      name: quay.io/thoth-station/s2i-pytorch-py38-notebook:latest
+    importPolicy:
+      scheduled: true
+    name: "latest"
+
+# s2i-data-science-notebook
+---
+apiVersion: image.openshift.io/v1
+kind: ImageStream
+metadata:
+  labels:
+    opendatahub.io/notebook-image: "true"
+  name: s2i-generic-data-science-notebook
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - from:
+      kind: DockerImage
+      name: quay.io/thoth-station/s2i-generic-data-science-notebook:v0.0.2
+    name: "3.8"
```

Comment:
out of curiosity, why are retagging the image from `v0.0.2` to `3.8`?  It looks like this is to reflect py version but i worry that it could be a bit confusing (ie, this is v3.8 of 's2i-generic-data-science-notebook)

Reply:
yeah cant we name it something like `s2i-generic-data-science-notebook-python38:v.0.0.2`

Reply:
i just saw the list of image which shows up on the dropdrop on idh jupyterhub 
they had tag of 3.6 to denote python version , thought of following the same pattern

I can change it to latest or v0.0.2, if you feel thats more accurate.

Reply:
Made the change as suggested :)